simulator/Field.py

```diff
@@ -1,21 +1,58 @@
-#!/usr/bin/env python3
+
 """
 File:          Field.py
 Author:        Moungsung Im 
-Last Modified: Binit on 9/23
+Last Modified: Moungsung on 10/8
 """
 
 import Buttons
 
 
 class Field:
-	"""The Field class maintains the state of the simulated elements within the field"""
-	def __init__(self):
-		"""Setups concrete measurements known about the field
-
-		Save measurements known about the field to the class
-		here. For example, the walls and their positions/
-		lengths. Also initializes simulated elements within
-		the field, like the buttons.
-		"""
-		pass
+    """The Field class maintains the state of the simulated elements within the field"""
+    def __init__(self):
+        """Setups concrete measurements known about the field
+        Save measurements known about the field to the class
+        here. For example, the walls and their positions/
+        lengths. Also initializes simulated elements within
+        the field, like the buttons.
```

Comment:
Use the imported Buttons module in your next PR to initialize the Buttons maintained here.